### ISSUE TITLE
Incorrect Error Message

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tomoakin/RPostgreSQL#38]
```
What steps will reproduce the problem?

1. Run Postgres with some data in a table

2. Do dbGetQuery("query").  The query should return some rows from a table.

3. Get a bad error message once for each select "unrecognized PostgreSQL field 
type void (id: 2278) in column 0.

What is the expected output? What do you see instead?
The error message should not be there.

What version of the product are you using? On what operating system?
Postgres 9.1.  Running Mac OSX Lion R 2.14.1

Please provide any additional information below.
Simple fix. Add the lines

case VOIDOID:
    break;

after line 546 in RS-PostgreSQL.c.

Basically column 0 on every select is of type void.


```

Original issue reported on code.google.com by `ashwin.k...@gmail.com` on 14 Jan 2012 at 2:56
